src/main/pg/max7456.c

```diff
@@ -25,12 +25,21 @@
 #include "pg/pg.h"
 #include "pg/pg_ids.h"
 
-#include "max7456.h"
+#include "drivers/io.h"
+#include "drivers/bus_spi.h"
 
-PG_REGISTER_WITH_RESET_TEMPLATE(max7456Config_t, max7456Config, PG_MAX7456_CONFIG, 0);
+#include "max7456.h"
 
-PG_RESET_TEMPLATE(max7456Config_t, max7456Config,
-    .clockConfig = MAX7456_CLOCK_CONFIG_OC, // SPI clock based on device type and overclock state
-);
+PG_REGISTER_WITH_RESET_FN(max7456Config_t, max7456Config, PG_MAX7456_CONFIG, 0);
 
+void pgResetFn_max7456Config(max7456Config_t *config)
+{
+    config->clockConfig = MAX7456_CLOCK_CONFIG_OC; // SPI clock based on device type and overclock state
+#ifdef MAX7456_SPI_CS_PIN
```

Comment:
With respect to full configurability, I think we should start collecting these default defines in one header file (included after `common_fc_post.h`), so that we can avoid having to move them out of the parameter group functions one by one when we do the move to universal targets.

Reply:
Good idea. Let me try it.